README.md

```diff
@@ -1,13 +1,162 @@
-# tools
+# Tools for authoring and serving codelabs
+Marc Cohen, [mcohen.io](https://mcohen.io), [@marcacohen](https://twitter.com/user/marcacohen)
```

Comment:
@x1ddos I feel like you belong here more than me. Would it be ok if I include your info here as well?

Reply:
btw, I think that gif is too heavy. I'll try making a smaller one.